workspaces/arborist/package.json

```diff
@@ -82,7 +83,7 @@
     "test-env": [
       "LC_ALL=sk"
     ],
-    "timeout": "360",
+    "timeout": "720",
```

Comment:
My computer has security programs on it that make disk access painfully slow, this was the only way to get tests to pass locally for me.